### DISCUSSION TITLE
Missing Tailwind CSS Utility Classes min-w-max and min-w-min

### DISCUSSION THREAD

**Discussion by kubersonzbw** [tailwindlabs/tailwindcss#13027] [Help] [answered]
Where did the Tailwind CSS classes `min-w-min` and `min-w-max` go?

**Comment by wongjn** [comment]
Nowhere, they should be available:

https://github.com/tailwindlabs/tailwindcss/blob/780163c58a428c94191700ef4f5d819152406349/stubs/config.full.js#L685-L689

If the classes are not available in your project, consider checking any misconfiguration.

**Reply by kubersonzbw** [reply]
When I update from `"tailwindcss": "3.3.2"` to `"tailwindcss": "3.4.1"`, those classes disappear automatically and I'm not doing anything else, just updating

**Reply by kubersonzbw** [reply]
![2024-02-26_13-18](https://github.com/tailwindlabs/tailwindcss/assets/114589180/4a9640d1-abcf-4d8c-bfd6-4143206b0a09)

**Reply by wongjn** [reply, marked as answer]
Assuming `defaultTheme` is an `import`/`require()` of `tailwindcss/defaultTheme`, then as per #12300, `minWidth` is now a function. So to update your Tailwind configuration, you could consider:

- Configuring your custom `minWidth` values at `theme.extends.minWidth` (if you don't have an empty `presets` array):
  ```js
  module.exports = {
    theme: {
      extends: {
         minWidth: {
            // '0' is not needed since it is already included in default Tailwind configuration.
           '1/4': '25%',
           '1/2': '50%',
           '3/4': '75%',
            // 'full' is not needed since it is already included in default Tailwind configuration.
         },
  ```
  This means you don't have to worry about getting the values from `defaultTheme` at all.
- Calling `defaultTheme.minWidth` to get the values:
  ```js
  minWidth: (args) => ({
    // '0' is not needed since it would be included by the `defaultTheme.minWidth()` call.
    '1/4': '25%',
    '1/2': '50%',
    '3/4': '75%',
    // 'full' is not needed since it would be included by the following call.
    ...defaultTheme.minWidth(args),
  }),
  ```

**Reply by kubersonzbw** [reply]
Thank you :)
